### PR TITLE
[2.11] Revert "fix: route port"

### PR DESCRIPTION
**Pull request by Sgitario**
This reverts commit 21ef3c382db86394fed765eabb3b30eb8609baf1.

### CODE REVIEW

File: annotations/openshift-annotations/src/main/java/io/dekorate/openshift/decorator/AddRouteDecorator.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void visit(KubernetesListBuilder list) {
         .withName(config.getName())
         .endTo()
         .withNewPort()
-        .withNewTargetPort(port.getContainerPort())
+        .withNewTargetPort(port.getHostPort())
```
Comment (iocanel) [inline annotations/openshift-annotations/src/main/java/io/dekorate/openshift/decorator/AddRouteDecorator.java:65]:
```suggestion
        //A service should be consumed without requiring consumers to know the container port.
        //In the same spirit the route should not refer to container port (as the mapping needs to happen on Service level).
        .withNewTargetPort(port.getHostPort())
```

**Review by iocanel — APPROVED** [review]
LGTM

Addded a suggestion: A comment that hopefully will prevent us from breaking this again.

**Comment by Sgitario** [comment]
We were wrong about reverting this change. From OpenShift docs:
`targetPort is the target port on pods that is selected by the service that this route points to.`

So, we should keep using the container port, not the host-port.

**Comment by Sgitario** [comment]
By the way, this was spotted by one test we have in our CI :)

**Pull request closed without merging**